src/searchForm/SearchForm.tsx

```diff
@@ -103,9 +103,8 @@ export const SearchForm = ({
   datepickerProps,
   stepperProps,
 }: SearchFormProps) => {
-  // Use React.useContext syntax so we can mock it
-  // https://github.com/enzymejs/enzyme/issues/2176#issuecomment-533582429
-  const mediaSize = React.useContext(MediaSizeContext)
+  const isLargeMediaSize = useIsLargeMediaSize()
+  const isSmallMediaSize = !isLargeMediaSize
```

Comment:
nit: 
```suggestions
const isSmallMediaSize = useIsSmallMediaSize()

Reply:
Why do we need a hook for this? 🤔

Reply:
How I see it: `useIsSmallMediaSize` already exists so I thought we should use it since it already abstracts the knowledge on whether it's small or not. Here we're "duplicating" the logic by saying `isSmallMediaSize = !isLargeMediaSize`. But this logic is so small that I'm okay to keep it as it is that's why it's really really really nitpicking 🙂

Reply:
I understand, my question was actually "Why did we create the hook in the first place?".
I think I'll keep it like this because I don't think this abstraction is useful.

Reply:
Ahhh sorry, then I'm guessing it's for readability & flexibility (in case we have a `medium` size in the future which I'm doubting) but I agree it's not super useful and maybe a bit overkill